src/NJsonSchema/JsonSerializerSettings.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace NJsonSchema.CodeGeneration.CSharp
+{
+    public class JsonSerializerSettings
+    {
+        public Newtonsoft.Json.DateParseHandling? DateParseHandling { get; set; }
+
+        public string DateFormatString { get; set; }
+
+        override
+        public string ToString() {
+            var settingsString = DateParseHandling == null ? 
+                                    string.Empty : 
+                                    $"DateParseHandling = {DateParseHandling.ToString()}";
+            
+            settingsString += string.IsNullOrEmpty(DateFormatString) ? 
+                                    string.Empty : 
+                                    $"DateFormatString = {DateFormatString}";
```

Comment:
I think we need to escape here with `\"`, eg

      $"DateFormatString = \"{DateFormatString}\"";